.github/workflows/build.yml

```diff
@@ -56,7 +56,13 @@ jobs:
       - name: Run build - apks
         working-directory: ./build
         run: cmake --build .
+      - name: Run tests
+        working-directory: ./build
+        run: >
+          ctest -C ${{matrix.build-type}}
```

Comment:
I added this even for android. Right now it runs 0 tests as they are all set to require hardware. This is here justs in case in the future we want to have host only tests or something like that